source/features/show-whitespace.css

```diff
@@ -41,15 +41,26 @@
 	--tab-size: 6;
 }
 
-/* File blocks in comments always have [data-tab-size=8] #4833 */
-[data-tab-size='8']:not(.highlight) {
+[data-tab-size='7'] {
+	--tab-size: 7;
+}
```

Comment:
Does this exist? Preferences don't have it

Reply:
You're right, this was more like (unnecessary) future-proofing 😂 

<img width="135" alt="Screenshot" src="https://user-images.githubusercontent.com/1402241/230862651-ec30b2fb-c11e-4ee0-89a9-5afc124c1b06.png">

Reply:
I will add this to lint (to-do)